docs/src/main/asciidoc/websockets-next-reference.adoc

```diff
@@ -661,6 +661,47 @@ class MyBean {
 ----
 <1> An asynchronous observer method is executed using the default blocking executor service.
 
+[[server-tenants]]
+==== Tenants
+Websockets-next cannot access `RoutingContext`. However, it can be done via `@Inject WebSocketConnection connection; connection.handshakeRequest().header()`.
```

Comment:
_"The `RoutingContext` is not available after the HTTP upgrade. However, it is possible to inject the `WebSocketConnection` and access the headers of the initial HTTP request."_

Reply:
+1